common/interfaces/user_interface/ui_input_text.c

```diff
@@ -498,11 +515,16 @@ static void next_btn_event_handler(lv_obj_t *next_btn, const lv_event_t event) {
       }
       break;
     case LV_EVENT_CLICKED:
+      /* TODO: Update after refactor*/
+      /* Below statement sets the entered text to a global variable.
+         Correct action: A pointer should be provided by application to
+         fill the entered text to and it should be populated here, and
+         Dummy string should be replaced with user provided pointer */
       mark_input(data->entered_text);
       if (ui_mark_list_choice)
```

Comment:
Are these 2 statements required?

Reply:
resolving since the statements are removed